types/mssql/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for mssql 4.3.0
+// Type definitions for mssql 6.0.0
 // Project: https://www.npmjs.com/package/mssql
 // Definitions by: COLSA Corporation <http://www.colsa.com/>
 //                 Ben Farr <https://github.com/jaminfarr>
```

Comment:
Can you remove my name in this pull request too?

I've changed jobs and haven't used mssql for awhile.

Reply:
Sure, it's been removed